examples/scenarioDeployingSolarArrays.py

```diff
@@ -0,0 +1,724 @@
+array. The array element frames are shifted outwards to each array's deployed center of mass location.
+
+The script is found in the folder ``basilisk/examples`` and executed by using::
+
+    python3 scenarioDeployingSolarArrays.py
+
+The scenario outputs eight plots. The first two plots illustrate the array 1 element angles and angle rates
+relative to the hub; while the following two plots illustrate the element angles and rates for the second array. The
+final plots illustrate the hub's inertial motion during the array deployment. The hub's inertial position, attitude,
+angular velocity, and angular velocity magnitude are given. This scenario also creates a Vizard simulation to
+visualize the solar array deployment.
+
+Illustration of Simulation Results
+----------------------------------
+
+The following plots illustrate the solar array deployment scenario simulation results.
+
+.. image:: /_images/Scenarios/scenarioDeployingSolarArrays_Array1ElementTheta.svg
+   :align: center
+
+.. image:: /_images/Scenarios/scenarioDeployingSolarArrays_Array2ElementTheta.svg
+    :align: center
+
+.. image:: /_images/Scenarios/scenarioDeployingSolarArrays_Array1ElementThetaDot.svg
+    :align: center
+
+.. image:: /_images/Scenarios/scenarioDeployingSolarArrays_Array2ElementThetaDot.svg
+    :align: center
+
+.. image:: /_images/Scenarios/scenarioDeployingSolarArrays_HubInertialPosition.svg
+    :align: center
+
+.. image:: /_images/Scenarios/scenarioDeployingSolarArrays_HubInertialMRPAttitude.svg
+    :align: center
+
+.. image:: /_images/Scenarios/scenarioDeployingSolarArrays_HubInertialAngularVelocity.svg
+    :align: center
+
+.. image:: /_images/Scenarios/scenarioDeployingSolarArrays_HubInertialAngularVelocityNorm.svg
+    :align: center
+
+"""
+
+#
+#   Solar Array Deployment Scenario
+#   Author:             Leah Kiner
+#   Creation Date:      March 4, 2024
+#
+
+import inspect
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+from Basilisk.architecture import messaging
+from Basilisk.simulation import spacecraft
+from Basilisk.simulation import prescribedMotionStateEffector
+from Basilisk.simulation import prescribedRotation1DOF
+from Basilisk.utilities import SimulationBaseClass
+from Basilisk.utilities import macros, RigidBodyKinematics as rbk
+from Basilisk.utilities import vizSupport
+
+filename = os.path.basename(os.path.splitext(__file__)[0])
+path = os.path.dirname(os.path.abspath(filename))
+
+def run(show_plots):
+    """
+    The scenario can be run with the followings set up parameter:
+
+    Args:
+        show_plots (bool): Determines if the script should display plots
+
+    """
+
+    simProcessName = "simProcess"
+    dynTaskName = "dynTask"
+    fswTaskName = "fswTask"
+
+    # Create a sim module as an empty container
+    scSim = SimulationBaseClass.SimBaseClass()
+
+    # Create test thread
+    dynTimeStep = 0.1  # [s]
```

Comment:
To make simulation run faster, you can increase dynamics and FSW time steps from 0.1 to 2.0s

Reply:
Done